On row action docs (#2631)
adobe/react-spectrum: packages/@react-aria/table/docs/useTable.mdx

`useTable` may be used in use cases where users can perform actions on rows or cells, such as navigating into items to open them or get more details. In the default `"toggle"` selection behavior, it is recommended to use a link in the first column for navigation.

In the `"replace"` selection behavior, the `onRowAction` and `onCellAction` prop can be used to enable row and cell actions, which differ depending on the interaction method. When provided to `useTableState`, double clicking with a mouse or pressing the <kbd>Enter</kbd> key triggers the action, while single click and the <kbd>Space</kbd> key are reserved for selection. On touch devices, the action becomes the primary tap interaction, and a long press enters into selection mode, which temporarily swaps the selection behavior to `"toggle"` to perform selection (you may wish to display checkboxes when this happens). Deselecting all items exits selection mode and reverts the selection behavior back to `"replace"`. Double clicking matches the behavior of desktop platforms like macOS and Windows, and a separate selection mode matches mobile platforms like iOS and Android.

> You'll recall that in the implementation of [Example](#example) above, we passed all props to `useTableState`, so we can just pass `onRowAction` through our `PokemonTable`.

Reviewer: mega nit
```suggestion
You'll recall that in our [table implementation](#example) above, we passed all props to `useTableState`, so we can just pass `onRowAction` through our `PokemonTable`.
```

Should there be an example of `onCellAction` too?
